libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DhWholesaleStartComponent } from './dh-wholesale-start.component';
+
+describe('DhWholesaleStartComponent', () => {
+  let component: DhWholesaleStartComponent;
+  let fixture: ComponentFixture<DhWholesaleStartComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [DhWholesaleStartComponent],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DhWholesaleStartComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
suggestion: this spec file is auto-generated. Unless there are tests that actually test something, it's better to delete it because it doesn't do much.

Reply:
will remove the file